src/sdg/producten/admin/product.py

```diff
@@ -38,15 +38,25 @@ class GeneriekProductAdmin(admin.ModelAdmin):
         "doelgroep",
         "verplicht_product",
         "verantwoordelijke_organisatie",
+        "eind_datum",
+        "product_status",
+        "is_sdg_product",
     )
     list_filter = (
         "doelgroep",
         "verplicht_product",
     )
     inlines = (LocalizedGeneriekProductInline,)
+    readonly_fields = ("product_status", "is_sdg_product")
     autocomplete_fields = ("verantwoordelijke_organisatie", "upn")
     search_fields = ("upn__upn_label",)
 
+    # Turns boolean into icon
+    def is_sdg_product(self, obj):
+        return obj.is_sdg_product
+
+    is_sdg_product.boolean = True
```

Comment:
This line can also be added directly in the model, but I prefer it this way to keep admin-related stuff in `admin.py`